app/models/like.rb

```diff
@@ -0,0 +1,4 @@
+class Like < ApplicationRecord
+  belongs_to :post, class_name: 'Post'
+  belongs_to :author_id, class_name: 'User'
```

Comment:
- [ ] Kindly change this code from `author_id` to `author` as an entity. A like should belong to an author (the like creator, not the ID).